dsl/pace/dsl/stencil_config.py

```diff
@@ -172,11 +171,13 @@ class StencilConfig(Hashable):
     dace_config: Optional[DaceConfig] = None
 
     def __post_init__(self):
-        self.backend_opts = self._get_backend_opts(
-            self.compilation_config.device_sync, self.compilation_config.format_source
-        )
+        self.backend_opts = {
+            "device_sync": self.compilation_config.device_sync,
+            "format_source": self.compilation_config.format_source,
+        }
```

Comment:
This is a good idea. Though I'm a little confused now from looking at `_get_backend_opts`, is there an expected value for the gpu backend? It looks like we've been running with `device_sync=True` for our runs lately.

Reply:
Question for myself: does this affect the `dace:*` backends. The runs so far seem to have used `device_sync=True`. We think this should not be a namelist option.

Reply:
> does this affect the dace:* backends?

It adds a `device_sync` call the way other GPU backends do.